packages/ui/src/Colors.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Grid } from './Grid';
+import { Technical } from './Typography';
```

Comment:
question: are these changes coming from #1489? It confused me since Technical and other components were not reviewed yet but the questions are mostly about them, not the color stories 😅

It's just that even if we merge this PR now, these changes will need to be reviewed once again with other changes

Reply:
@VanishMax ah, yes, but I haven't made that PR ready for review yet (oops). My plan was to wait until they're all approved, then merge them into each other and finally into `main`. If you give this the 👍🏻 , I'll wait to merge it until #1489 is approved and merged.

Reply:
Just marked #1489 ready for review as well